libvast/vast/defaults.hpp

```diff
@@ -78,6 +78,10 @@ struct json {
   /// Nested category in config files for this subcommand.
   static constexpr const char* category = "import.json";
 
+  /// Time that the reader waits for new data before it finishes a partial slice.
+  static constexpr std::chrono::milliseconds partial_slice_read_timeout
+    = std::chrono::milliseconds{500};
+
```

Comment:
I don't think we need to define this default for each reader individually, so we could just have one in the `import::shared` namespace.

Reply:
Fair point.